frontend/src/components/IstioWizards/WizardAnnotations.tsx

```diff
@@ -188,14 +189,15 @@ export class WizardAnnotations extends React.Component<Props, State> {
     const header = (
       <>
         <Title id="modal-custom-header-label" headingLevel="h1" size={TitleSizes['2xl']}>
-          {this.props.canEdit ? 'Edit ' : 'View '}annotations
+          {this.props.canEdit ? 'Edit ' : 'View '}
+          {this.props.header ? this.props.header : 'annotations'}
```

Comment:
I don't think annotations should be default value (see my comment above). I would make header to be mandatory and don't set default value.